### PR TITLE
Add bounces apiblueprint

### PR DESCRIPTION
**Pull request by jovel**
Ticket: MAKO-3603

Matching apiary: jsapi

(edited by mbernier to remove URLS to resources)

### CODE REVIEW

File: source/API_Reference/Web_API_v3/bounces.apiblueprint [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+FORMAT: 1A
+
+# Bounces
+
+
+## Bounces Collection [/v3/suppression/bounces?start_time={start_time}&end_time={end_time}]
+
++ Parameters
+
+    + start_time (optional, number, `1443651141`) ... Refers start of the time range when a bounce was created.
```
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: Im assuming this is a unix timestamp. Do we normally specify that?

File: source/API_Reference/Web_API_v3/bounces.apiblueprint [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    + end_time (optional, number, `1443651154`) ... Refers end of the time range when a bounce was created.
```
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: Are the start and end time inclusive or exclusive?

File: source/API_Reference/Web_API_v3/bounces.apiblueprint [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+There's two bounce delete options:
+1) deleting all bounces by specifying "delete_all" to true in the request body, or
+2) deleteing some emails by specificy "emails" and an array of emails in the response body.
+
+### Delete bounces [DELETE]
+
++ Request (application/json)
+
+        {
```
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: You missing a `+ Body`. This should be in the following format:

`+ Request (application/json)

    + Body

            {
                "delete_all": false
            }
`

File: source/API_Reference/Web_API_v3/bounces.apiblueprint (hunk 2 above)
Comment (jovel) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: So it's marked as timestamp in apiary but something in the docs compiler throws up when I write timestamp so now it's number.

File: source/API_Reference/Web_API_v3/bounces.apiblueprint [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            "delete_all": true
+        }
+
++ Response 204
+
++ Request (application/json)
+
+        {
```
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: Missing `+ Body` here also

File: source/API_Reference/Web_API_v3/bounces.apiblueprint (hunk 1 above)
Comment (jovel) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: So it's marked as timestamp in apiary but something in the docs compiler throws up when I write timestamp so now it's number.

File: source/API_Reference/Web_API_v3/bounces.apiblueprint (hunk 2 above)
Comment (jovel) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: As far as I can tell, it should be inclusive

File: source/API_Reference/Web_API_v3/bounces.apiblueprint [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        }
+
++ Response 204
+
+## Bounces Item [/v3/suppression/bounces/{email}]
++ Parameters
+
+    + email ('required', string, `bounce1@test.com`) ... Email address of bounce entry to retrieve
+
+### Get a bounce [GET]
```
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: When tested locally Im seeing a DELETE in the request url for this section. Not sure why though.

```DELETE https://api.sendgrid.com/v3/v3/suppression/bounces/{email} HTTP/1.1```

File: source/API_Reference/Web_API_v3/bounces.apiblueprint [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+        [
+            {
+                "created": 1443651125,
+                "email": "bounce1@test.com",
+                "reason": "550 5.1.1 The email account that you tried to reach does not exist. Please try double-checking the recipient's email address for typos or unnecessary spaces. Learn more at  https://support.google.com/mail/answer/6596 o186si2389584ioe.63 - gsmtp ",
+                "status": "5.1.1"
+            }
+        ]
+### Delete a bounce [DELETE]
```
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: I'm not seeing this header when testing locally

File: source/API_Reference/Web_API_v3/bounces.apiblueprint (hunk 5 above)
Comment (dmowcomber) [inline source/API_Reference/Web_API_v3/bounces.apiblueprint]: Nevermind. This section looks good now after pulling your latest changes

**Comment by dmowcomber** [comment]
LGTM. Tested locally

**Comment by mbernier** [comment]
@jovel @dmowcomber Is this ready to go live?

**Comment by jovel** [comment]
@mbernier Yup, this doc is ready to go live if you're ok with the wording. The actual endpoint has been up since makoapi was first in beta.

**Pull request merged**